patterns/pe.hexpat

```diff
@@ -104,9 +148,31 @@ struct DOSStub {
 	u8 data[11];
 };
 
+union SectionMisc {
+	u32 physicalAddress;
+	u32 virtualSize;
+};
+
+struct Section {
+	s8 name[8];
+	SectionMisc misc;
+	u32 virtualAddress;
+	u32 sizeOfRawData;
+	u32 ptrRawData;
+	u32 ptrRelocations;
+	u32 ptrLineNumbers;
+	u16 numberOfRelactions;
+	u16 numberOfLineNumbers;
+	u32 characteristics;
+};
+
 struct PEHeader {
 	DOSHeader dosHeader;
 	DOSStub dosStub;
 };
 
-PEHeader peHeader @ 0x00;
\ No newline at end of file
+PEHeader peHeader @ 0x00;
+
+#define COFF_HEADER peHeader.dosHeader.coffHeaderPointer
+
+Section sectionsTable[COFF_HEADER.numberOfSections] @ nextAfter("COFF_HEADER");
```

Comment:
oof this looks so weird but works because my preprocessor isn't super sophisticated. Maybe put the `"` into the define as well. That will make it look more natural

Reply:
Unfortunately I actually can't because I would not be able to access the numberOfSections field that way. I'm just gonna delete the define, the string isn't very long anyway.